projects/longest-word-in-string/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Find length of the longest word in Every Language
+layout: default
+date: 2020-10-25
+last-modified: 2020-10-25
+featured-image:
+tags: [longest-word-in-string, javascript]
+authors:
+    - barhouum7
+---
+
+In this article, we'll outline the longest word in a string project.
+
+## Requirements
+
+The purpose of this program is to return the length of the longest word in the provided sentence as follows:
+
+```javascript
+longest-word-in-string.lang "Google do a barrel roll"
+Output: "6"
+```
+
+The result should be a number.
+
+## Testing
+
+The following table contains various test cases that you can use to verify the 
+correctness of your solution:
+
+| Description | Input | Output |
+|--------------|-------|--------|
+| No Input | | "Usage: please provide a string" |
+| Empty Input | "" | "Usage: please provide a string" |
+| Simple Input | "May the force be with you" | "5" |
```

Comment:
This looks good! Perhaps we could add at least one more test case for a single word. Also, just like your other documentation, we need to define what a **word** is.

Reply:
Please check the last changes